### PR TITLE
don't  redo everything every time + workaround for due date of milestone

### PR DESCRIPTION
**Pull request by zaibon**

### CODE REVIEW

File: lib/JumpScale/clients/github/Repository.py [hunk 1]
```diff
@@ -323,13 +323,12 @@ def getBody(descr, name, owner):
...
-            due = j.data.time.epoch2pythonDateTime(
-                int(j.data.time.any2epoch(deadline)))
+            due = j.data.time.epoch2pythonDateTime(int(j.data.time.any2epoch(deadline)))
             self.logger.info("Create milestone on %s: %s" % (self, title))
             body = getBody(description.strip(), name, owner)
-            # @todo cannot set deadline, please fix. See https://github.com/PyGithub/PyGithub/issues/396
-            self.api.create_milestone(
-                title=title, description=body)#, due_on=due)
+            # workaround for https://github.com/PyGithub/PyGithub/issues/396
+            milestome = self.api.create_milestone(title=title, description=body)  # , due_on=due)
```
Comment (rkhamis) [inline lib/JumpScale/clients/github/Repository.py]: "milestome" typo

File: lib/JumpScale/clients/github/Issue.py [hunk 2]
```diff
@@ -419,61 +432,69 @@ def add_task(self, task):
...
+        doc = j.data.markdown.getDocument(self.body)
+        i = 0
+
+        for token in doc.tokens:
+            if token['type'] == 'table':
+                task_table_found = True
+                existing_tasks = [c[2] for c in token['cells']]
+                if "#%d" % task.number not in existing_tasks:
+                    change = True
+                    # TODO add issue to tasks
```
Comment (FastGeert) [inline lib/JumpScale/clients/github/Issue.py]: Why add this todo, and not implement it?

Comment (zaibon) [inline lib/JumpScale/clients/github/Issue.py]: The line 455 does it. This comment should be removed. My bad

File: lib/JumpScale/clients/github/Repository.py [hunk 3]
```diff
@@ -442,7 +441,7 @@ def get_story(name):
                 try:
                     cmd, args = todo.split(' ', 1)
                 except Exception as e:
-                    print ("*** WARNING:%s, cannot process todo for %s"%(str(e),todo))                    
+                    print("*** WARNING:%s, cannot process todo for %s" % (str(e), todo))
```
Comment (FastGeert) [inline lib/JumpScale/clients/github/Repository.py]: Why use print instead of self.logger.warn

**Comment by zaibon** [comment]
obsolete, resolved by :https://github.com/Jumpscale/jumpscale_core8/commit/137be032da2784c9ce51b737e0ba0db0e6e8edc7

**Pull request closed without merging**